config/jobs/kubernetes/sig-cloud-provider/gcp/gce-conformance.yaml

```
@@ -37,6 +37,170 @@ periodics:
         requests:
           cpu: 1
           memory: 3Gi
+      env:
+      # cluster/gce/gci/configure-kubeapiserver.sh allows us to override
+      # the default audit-policy
+      # We need to remove the events filter to ensure we can audit
+      # conformance tests are actually hitting event endpoint operations
+      # Ideally we could patch the audit policy within this job definiton
+      # instead of overriding the policy via ADVANCED_AUDIT_POLICY
```

Maybe this is too much context that instead belongs in the commit message? But this would be my suggestion:
```suggestion
      # cncf/apisnoop needs to consume audit logs from the cluster under test in order to
      # effectively report on conformance test coverage of api operations. The default audit
      # policy excludes events from being logged due to performance reasons. For this
      # job we will accept the performance hit and allow events to be logged.
      # 
      # We do this by setting ENABLE_APISERVER_ADVANCED_AUDIT which tells 
      # cluster/gce/gci/configure-kubeapiserver.sh to override the default audit policy with the
      # contents of the ADVANCED_AUDIT_POLICY env var.  We've copy-pasted the default
      # audit policy as output by https://github.com/kubernetes/kubernetes/blob/b1e3a2ac7a80d49ef6153457eaa35dac4fc98444/cluster/gce/gci/configure-helper.sh#L1077-L1235
      # and then removed the block that says "don't log events"
      # 
      # NOTE: if the default audit policy changes, this will need to be updated accordingly
      # TODO(a-link-to-a-github-issue): ideally we could do this via a patch instead
```